src/index.less

```diff
@@ -0,0 +1,256 @@
+      background: @chatbot-header-bg-color1;
+      background: radial-gradient(circle at bottom left, @chatbot-header-bg-color1, @chatbot-header-bg-color2);
+
+      button {
+        display: none;
+      }
+
+      h6 {
+        color: #fff;
+        position: relative;
+        margin: 0;
+        font-size: @chatbot-header-font-size;
+      }
+    }
+
+    .chatbot-body {
+      overflow-x: hidden;
+      overflow-y: auto;
+      padding: 17px 17px 0 17px;
+      height: ~'calc(100% - @{chatbot-header-height} - @{chatbot-footer-height})';
+
+      .chatbot-message {
+        display: flex;
+        position: relative;
+        flex-direction: column;
+        flex-shrink: 0;
+        margin-top: 35px;
+
+        .chatbot-avatar {
+          position: absolute;
+          top: -18px;
+          left: -43px;
+          float: left;
+          flex-shrink: 0;
+          width: 32px;
+          height: 34px;
+          margin-right: 10px;
+        }
+
+        .chatbot-time {
+          color: @chatbot-time-color;
+          font-size: 14px;
+          position: absolute;
+          top: -20px;
+        }
+
+        .chatbot-text {
+          display: inline-block;
+          position: relative;
+          padding: 12px 14px;
+        }
+      }
+
+      .chatbot-message-user {
+        .chatbot-time {
+          right: 0;
+        }
+
+        .chatbot-text {
+          background-color: @chatbot-user-message-bg-color;
+          border-radius: 12px 12px 0 12px;
+          color: @chatbot-user-message-color;
+          margin-left: auto;
+        }
+      }
+
+      .chatbot-message-bot {
+        margin-left: 43px;
+
+        .chatbot-text {
+          background-color: @chatbot-bot-message-bg-color;
+          border-radius: 0 12px 12px 12px;
+        }
+      }
+    }
+
+    .chatbot-footer {
+      position: absolute;
+      bottom: 0;
+      width: 100%;
+      background-color: @chatbot-bg-color;
+
+      form {
+        position: relative;
+        margin: @chatbot-form-margin;
+
+        textarea {
+          display: block;
+          border-radius: 25px;
+          border: @chatbot-textarea-border-size solid @chatbot-textarea-border-color;
+          box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.1);
+          height: auto;
+          line-height: @chatbot-textarea-line-height;
+          padding: @chatbot-textarea-line-height;
+          padding-right: @chatbot-textarea-line-height + @chatbot-send-button-size;
+          overflow: hidden;
+          resize: none;
+          outline: none;
+          width: 100%;
+        }
+
+        button {
+          position: absolute;
+          bottom: 18px;
+          right: 18px;
+          width: @chatbot-send-button-size;
+          height: @chatbot-send-button-size;
+
+          svg {
+            width: @chatbot-send-button-size;
+            height: @chatbot-send-button-size;
+          }
+        }
+      }
+    }
+  }
+
+  @media @chatbot-smallscreen {
```

Comment:
Could you mobile first ? :D

Reply:
??

Reply:
I think @FredericEspiau suggest to write Mobile-first style (vs Desktop-first style) and adapt styles in media queries for larger screen (here, the styles is written for Desktop-first and adapted for smaller screens)

Reply:
Yup :)